### PR TITLE
device logs cleanup

### PR DESCRIPTION
**Pull request by snopoke**

### CODE REVIEW

**Review by emord — APPROVED** [review]
I can deploy this today

File: corehq/ex-submodules/phonelog/tasks.py [hunk 1]
```diff
@@ -2,10 +2,13 @@
...
+from phonelog.models import DeviceReportEntry, UserErrorEntry, ForceCloseEntry, UserEntry
 
 
 @periodic_task(run_every=crontab(minute=0, hour=0), queue=getattr(settings, 'CELERY_PERIODIC_QUEUE', 'celery'))
 def purge_old_device_report_entries():
     max_age = datetime.utcnow() - timedelta(days=60)
     DeviceReportEntry.objects.filter(server_date__lt=max_age).delete()
+    UserErrorEntry.objects.filter(server_date__lt=max_age).delete()
+    ForceCloseEntry.objects.filter(server_date__lt=max_age).delete()
+    UserEntry.objects.filter(server_date__lt=max_age).delete()
```
Comment (gcapalbo) [inline corehq/ex-submodules/phonelog/tasks.py]: i think this won't delete the records with server_date null, is that intended?

Comment (emord) [inline corehq/ex-submodules/phonelog/tasks.py]: I think that makes sense. This made the migration really fast. we can fill those with today's date at a later time.

Comment (gcapalbo) [inline corehq/ex-submodules/phonelog/tasks.py]: ok, could even wait a little while and delete all the ones with nulls

**Comment by snopoke** [comment]
Yes that was what I was intending to do.

**Pull request merged**